Source/VMware.PSDesiredStateConfiguration/VMware.PSDesiredStateConfiguration.psd1

```diff
@@ -21,7 +21,7 @@ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
 ModuleToProcess = 'VMware.PSDesiredStateConfiguration.psm1'
 
 # Version number of this module.
-ModuleVersion = '0.0.0.1'
+ModuleVersion = '1.0.0.1'
```

Comment:
I think this shouldn't be changed now, we will update it when we release it, for now is ok to be 0.0.0.2 after the changed is merged, which will be done by the build procedure.